src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -123,19 +132,31 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
     } else {
       const parsed = Math.floor(Number.parseFloat(value) * 100)
 
-      if (!Number.isInteger(parsed) || parsed < 0 || parsed > 5000) {
+      if (!Number.isInteger(parsed) || parsed < MIN_SLIPPAGE_BPS || parsed > MAX_SLIPPAGE_BPS) {
+        ReactGA.event({
+          category: 'Order Slippage Tolerance',
+          action: 'Set Default Slippage Tolerance',
+          value: DEFAULT_SLIPPAGE_BPS,
+        })
         setUserSlippageTolerance('auto')
         if (value !== '.') {
           setSlippageError(SlippageError.InvalidInput)
         }
       } else {
+        ReactGA.event({
+          category: 'Order Slippage Tolerance',
+          action: 'Set Custom Slippage Tolerance',
+          value: parsed,
+        })
```

Comment:
Should probably set a debounce or submit to GA only after the modal is closed, otherwise every keystroke will send a new event, and we won't have a very accurate picture

Also, clicking on Auto doesn't trigger a GA event

https://user-images.githubusercontent.com/43217/170958683-aa53bdf1-1ff8-48ad-9c94-dc706370d16a.mov

Reply:
I would suggest the same as Leo or you can consider debouncing

Reply:
Fixed the two issues. Now we debounce 2s, also clicking in `auto` works